src/sortedDict.jl

```diff
@@ -256,22 +260,55 @@ immutable SDKeyIteration{T <: SDIterableTypesBase}
     base::T
 end
 
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{SortedDict{K,D,Ord}}) = K
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{ExcludeLast{K,D,Ord}}) = K
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{IncludeLast{K,D,Ord}}) = K
```

Comment:
These can be simplified to
```julia
eltype{K}(s::SDKeyIteration{SortedDict{K}}) = K
eltype{K}(s::SDKeyIteration{ExcludeLast{K}}) = K
eltype{K}(s::SDKeyIteration{IncludeLast{K}}) = K
```

Actually, though, did you check that these definitions did the right thing?  In the past, I believe the parser ignored the inner set of parenthesis, although that might not be the case any more.